label_studio/projects/api.py

```diff
@@ -572,6 +572,23 @@ def perform_create(self, serializer):
         return instance
 
 
+class ProjectGroundTruthTaskListAPI(ProjectTaskListAPI):
+    '''
+    Same as ProjectTaskListAPI with the exception that this API only returns tasks
+    that contain at least one ground truth annotation
+    '''
+
+    def filter_queryset(self, queryset):
+        project = generics.get_object_or_404(Project.objects.for_user(self.request.user), pk=self.kwargs.get('pk', 0))
+        ground_truth_query = Q(annotations__was_cancelled=False) & Q(annotations__result__isnull=False) & Q(annotations__ground_truth=True)
+        tasks = Task.objects.filter(project=project).filter(ground_truth_query).order_by('-updated_at')
```

Comment:
Probably just for by own understanding, but is the "-" in `.order_by('-updated_at')` for reversing the order or something?

Reply:
Correct 👍🏼